helm-charts/example-apiserver/templates/konkservice.yaml

```diff
@@ -26,5 +26,3 @@ spec:
   {{- with .Values.konkservice }}
   {{- toYaml . | nindent 2 }}
   {{- end }}
-  crds: |
-{{ (.Files.Glob "crds/*").AsConfig | indent 4 }}
```

Comment:
👍

Reply:
We can also remove the `crds` directory as well.

Reply:
I think I'll move this to another PR to simplify this one.

https://github.com/infobloxopen/konk/pull/102